CONTRIBUTING.md

```diff
@@ -46,8 +47,16 @@ accept your pull requests.
 
 1.  Run the tests:
 
+        # Run all tests.
         npm test
 
+        # Run all unit tests.
+        npm run test-only
```

Comment:
@JustinBeckwith @bcoe what do we do when this isn't true?

Reply:
@stephenplusplus yeah, I missed when this landed that the npm commands aren't quite the canonical commands we have in all our libraries. Want to take a stab at updating here:

https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/CONTRIBUTING.md